src/analytics/formTracker/formTracker.ts

```diff
@@ -123,13 +121,14 @@ export class FormTracker extends BaseTracker {
   getFieldType(elements: FormField[]): string {
     if (elements[0].type === "select-one") {
       return this.DROPDOWN_FIELD_TYPE;
-    } else if (elements[0].type === "radio") {
+    }
+    if (elements[0].type === "radio") {
       return this.RADIO_FIELD_TYPE;
-    } else if (elements[0].type === "checkbox") {
+    }
+    if (elements[0].type === "checkbox") {
       return elements[0].type;
-    } else {
-      return this.FREE_TEXT_FIELD_TYPE;
     }
+    return this.FREE_TEXT_FIELD_TYPE;
```

Comment:
Yeah I must admit I do prefer this to `if else` just because I find trying to stack up in my head what `else` conditions have already been considered while I read through. This must have been an autofix because I don't remember changing it, but I'm not sure what the rule was.

Either way though, agree that `switch` is better than either of these, so I've changed it to that.